### DISCUSSION TITLE
Print traceback without borders

### DISCUSSION THREAD

**Discussion by vladmandic** [Textualize/rich#2925] [Q&A]
I've search the docs, but can't seem to find how to disable borders when using `console.print_exception`?  

I like them visually, but they do result in a lot of copy&paste when people try to post a traceback to (for example) github issues.  

Best I could find so far is to paint them black, but that doesn't resolve original issue (painting with none does not work):

```python
console = Console(log_time=True, log_time_format='%H:%M:%S-%f', theme=Theme({
    "traceback.border": "black",
    "traceback.border.syntax_error": "black",
}))
```

Also, i don't see a style param that is used for "locals" borders at all?
Its just printed as yellow no matter what.

**Comment by mb6852** [comment]
Did a feature ever get added to turn off traceback border?

**Reply by TomJGooding** [reply]
It sounds like this may be added in a future release - see https://github.com/Textualize/rich/pull/2996#issuecomment-2200110998

**Comment by quartzar** [comment]
This is pretty much the only thing holding us back from using rich tracebacks right now - it just makes it horrendous to try copy&paste a traceback with all the border characters.
